### PR TITLE
Config and Vector Icons Setup Completed

### LINKED ISSUE #3: Integrate Config and Vector Icons (Ionicons)
Integrate react-native-config and vector icons

🕐 :- 4 to 5 hours

### PR DESCRIPTION
**Pull request by Naveen-gali**
- Resolve issue #3

### CODE REVIEW

File: package-lock.json
Comment (prajnab-mm) [inline package-lock.json:1]: Do we need this? We are using yarn.lock

File: package.json [hunk 1]
```diff
@@ -7,11 +7,16 @@
     "ios": "react-native run-ios",
     "lint": "eslint .",
     "start": "react-native start",
-    "test": "jest"
+    "test": "jest",
+    "build-stage-ios": "ENVFILE=.env.staging npx react-native run-ios -- --reset-cache",
+    "build-stage-android": "ENVFILE=.env.staging npx react-native run-android -- --reset-cache",
```
Comment (prajnab-mm) [inline package.json]: After run-android, "--" is that a mistake? as it has no option with it

Comment (Naveen-gali) [reply package.json]: Actually for having options they suggested to keep a `--` and use options like --reset-cache

File: package-lock.json
Comment (Naveen-gali) [reply package-lock.json:1]: dont know i will try removing that and will see it as we are not using npm it shouldn't cause any issues

**Review by prajnab-mm — APPROVED** [review]

**Pull request merged**